visibility.md

```diff
@@ -1,7 +1,70 @@
 # Code of Conduct visibility
 
-- Advise from our transparency report on how to make CoC visible
-- Guidance on what should be on the CoC page
-- Guidance on what should be on the posters
-- Guidance where to link to CoC page
+One of the most important things to do before and during the event is 
+making sure that everyone knows about existence of Code of Conduct and 
+have no problems to find details about it. Failing to communicate about it 
+might cause unnecessary discussions in case of problems or might discourage 
+some people from attending the event.
 
+Here is a list of things that help to increase visibility of the Code of Conduct:
+
+* reading and agreeing to your Code of Conduct should be obligatory when registering ticket for the event
+* make sure to link your CoC in both header and footer on your website
```

Comment:
let's emphasise here that footer is not enough. it's also worth to mention code of conduct on your homepage.